jdaviz/configs/specviz/plugins/line_analysis/line_analysis.vue

```diff
@@ -133,55 +133,57 @@
         ></v-progress-circular>
       </div>
 
-      <j-plugin-section-header>Redshift from Centroid</j-plugin-section-header>
-      <v-row>
-        <j-docs-link>Assign the centroid reported above to the observed wavelength of a given line and set the resulting redshift.  Lines must be loaded and plotted through the Line Lists plugin first.</j-docs-link>
-      </v-row>
-      <v-row class="row-no-outside-padding">
-        <v-col cols=2>
-          <j-tooltip tipid='plugin-line-analysis-sync-identify'>
-            <v-btn icon @click="() => sync_identify = !sync_identify" style="margin-top: 14px">
-              <img :class="sync_identify ? 'color-to-accent' : 'invert-if-dark'" :src="sync_identify ? sync_identify_icon_enabled : sync_identify_icon_disabled" width="20"/>
-            </v-btn>
-          </j-tooltip>
-        </v-col>
-        <v-col cols=10>
-          <v-select
-            :menu-props="{ left: true }"
-            attach
-            :items="line_items"
-            v-model="selected_line"
-            label="Line"
-            hint="Select reference line."
-            persistent-hint
-          ></v-select>
-        </v-col>
-      </v-row>
+      <div v-if="lines_loaded">
```

Comment:
looks like a separate traitlet might not be necessary

```suggestion
      <div v-if="line_items.length > 0">
```